junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedTestExtension.java

```diff
@@ -122,8 +122,8 @@ private ExtensionContext.Store getStore(ExtensionContext context) {
 	}
 
 	private TestTemplateInvocationContext createInvocationContext(ParameterizedTestNameFormatter formatter,
-			ParameterizedTestMethodContext methodContext, Object[] arguments) {
-		return new ParameterizedTestInvocationContext(formatter, methodContext, arguments);
+			ParameterizedTestMethodContext methodContext, Object[] arguments, Integer invocationIndex) {
```

Comment:
You could use an `int` here (same remark in `ParameterizedTestInvocationContext`, `ParameterizedTestMethodContext` and `ParameterizedTestParameterResolver`)